http/sys_health.go

```diff
@@ -34,32 +36,61 @@ func fetchStatusCode(r *http.Request, field string) (int, bool, bool) {
 }
 
 func handleSysHealthGet(core *vault.Core, w http.ResponseWriter, r *http.Request) {
+	code, body, err := getSysHealth(core, r)
+	if err != nil {
+		respondError(w, http.StatusInternalServerError, err)
+		return
+	}
+
+	if body == nil {
+		respondError(w, code, nil)
+		return
+	}
+
+	w.Header().Add("Content-Type", "application/json")
+	w.WriteHeader(code)
 
+	// Generate the response
+	enc := json.NewEncoder(w)
+	enc.Encode(body)
+}
+
+func handleSysHealthHead(core *vault.Core, w http.ResponseWriter, r *http.Request) {
+	code, body, err := getSysHealth(core, r)
+	if err != nil {
+		respondError(w, http.StatusInternalServerError, err)
```

Comment:
I don't think this can be used here -- `respondError` will write a body, which according to the RFC must not be done with `HEAD`. You probably need to simply write the status code into the header and return.